src/NewsService/Controllers/NewsController.cs

```diff
@@ -9,20 +10,28 @@ namespace LT.DigitalOffice.NewsService.Controllers
     [ApiController]
     public class NewsController : ControllerBase
     {
+        [HttpGet("getNewsById")]
+        public NewsResponse GetNewsInfoById(
+            [FromServices] IGetNewsByIdCommand getNewsByIdCommand,
+            [FromQuery] Guid newsId)
+        {
+            return getNewsByIdCommand.Execute(newsId);
+        }
+
         [HttpPost("editNews")]
         public void EditNews(
-            [FromServices] IEditNewsCommand command,
```

Comment:
Why you don't want to use the word "command"?

Reply:
That was good for my understanding, but okay, i will change it

Reply:
you can don't do it, I just asked

Reply:
too late, already change it